### DISCUSSION TITLE
Date of planned deployment of new verison

### DISCUSSION THREAD

**Discussion by DDDenisSobek** [Megabit/Blazorise#1952] [Q&A]
Hi everybody,
i want to ask, where i would see the date of deployment of a new version, if a deployment is planned?
I ask because, sometimes i want to use some feature of a new version, but i do not want to use the preview version.
If i know the date, i can plan my development and will not have to look up everyday.

**Comment by stsrki** [comment]
The release is in the process :)

My third world country upload speed is a joke so it takes time 🤦‍♂️

**Reply by stsrki** [reply]
It's going to be done during a day

**Reply by DDDenisSobek** [reply]
And in general?

**Reply by stsrki** [reply]
In general, it's just released! You're welcome! 😉
